### PR TITLE
feat: Implement changes in the Visioconference administration page - EXO-68226

### PR DESCRIPTION
**Pull request by IlhemEssaadi**

### CODE REVIEW

File: webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue [hunk 1]
```diff
@@ -1,86 +1,51 @@
...
+            class="providersSwitcher"
+            @change="changeActive(providerConfig)" />
+        </v-list-item-action>
+        <v-list-item-content class="py-0">
+          <v-list-item-title class="subtitle-1 pt-2">
+            {{ $t(`webconferencing.admin.${providerConfig.title}.name`)
+              ? $t(`webconferencing.admin.${providerConfig.title}.name`)
+              : providerConfig.title }} 
+          </v-list-item-title>
+          <div
```
Comment (rdenarie) [inline webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue]: Shoud we use
`<v-list-item-subtitle>` instead of div ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue [hunk 2]
```diff
@@ -1,86 +1,50 @@
...
+            @change="changeActive(providerConfig)" />
+        </v-list-item-action>
+        <v-list-item-content class="py-0">
+          <v-list-item-title class="subtitle-1 pt-2">
+            {{ $t(`webconferencing.admin.${providerConfig.title}.name`)
+              ? $t(`webconferencing.admin.${providerConfig.title}.name`)
+              : providerConfig.title }} 
+          </v-list-item-title>
+          <v-list-item-subtitle
+            v-html="$t(`webconferencing.admin.${providerConfig.title}.description`)
```
Comment (rdenarie) [inline webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue]: why using v-html here ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue (hunk 2 above)
Comment (IlhemEssaadi) [reply webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue]: > why using v-html here ?

For the inclusion of an HTML anchor element with a link found in the WebRTC connector description. However, as we're going to remove the webrtc connector, we can dispense with the use of v-html.

Comment (rdenarie) [reply webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue]: Yes, please remove the v-html

**Review by rdenarie — APPROVED** [review]

**Pull request merged**